pkg/github/github.go

```diff
@@ -171,13 +274,12 @@ func (g *GitHubClient) DeleteIssueComment(ctx context.Context, owner, repo strin
 }
 
 // ListIssueComments lists existing comments for an issue or pull request.
-func (g *GitHubClient) ListIssueComments(ctx context.Context, owner, repo string, number int, opts *github.IssueListCommentsOptions) ([]*github.IssueComment, *github.Response, error) {
+func (g *GitHubClient) ListIssueComments(ctx context.Context, owner, repo string, number int, opts *github.IssueListCommentsOptions) ([]*int64, error) {
```

Comment:
I dont know if I agree with this, we may want to filter based on comment body or author, etc.

Reply:
Perhaps we should add support for those when we need to? We can return a lightweight struct of the additional values when we determine what we need?

Reply:
Can we just create the struct now and just have the ID? That way we dont have to change types, just add fields? (same for GetIssueComment and GetIssue also)

Reply:
I agree with Mike

Reply:
Done